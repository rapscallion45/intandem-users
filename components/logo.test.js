import React from 'react';
import { shallow } from 'enzyme';
import { render } from '@testing-library/react';
import toJson from 'enzyme-to-json';
import Logo from './logo';

describe('Logo', () => {
  it('Renders correctly enzyme', () => {
    /* Arrange */
    /* Act */
    const wrapper = shallow(<Logo />);

    /* Assert */
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  describe('Logo Image', () => {
    it('Should render logo image from expected source', () => {
      /* Arrange */
      /* Act */
      const wrapper = render(<Logo />);
      const image = wrapper.getByAltText('logo-image');

      /* Assert */
      expect(wrapper.getByAltText('logo-image')).toBeInTheDocument();
      expect(image.src).toContain('/static/logo.webp');
    });
  });
});
