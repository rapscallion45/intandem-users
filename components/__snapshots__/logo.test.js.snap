// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Renders correctly enzyme 1`] = `
<Styled(ForwardRef(Box))
  alt="logo-image"
  component="img"
  src="/static/logo.webp"
  sx={Object {}}
/>
`;
